plugin/backend.go

```diff
@@ -109,7 +109,6 @@ func (pm *Manager) Push(name string, metaHeader http.Header, authConfig *types.A
 	if err != nil {
 		return err
 	}
-	defer config.Close()
```

Comment:
@dmcgowan
Thanks for your feedback.
Actually, `config`  and the following `rootfs` is Closed in the `distribution.Push` of `plugin/distribution/push.go`.